torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9181,7 +9189,11 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                    domain=(1, None),
                    dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
                    dtypesIfCUDA=all_types_and_complex_and(torch.chalf, torch.bool, torch.half, torch.bfloat16),
-                   decorators=(precisionOverride({torch.bfloat16: 5e-2}),),
+                   decorators=(precisionOverride({torch.bfloat16: 5e-2}),
+                               DecorateInfo(toleranceOverride({torch.complex64: tol(atol=1e-05, rtol=1e-05)}),
+                                            'TestUnaryUfuncs',
+                                            'test_reference_numerics_extremal',
+                                            device_type='cuda'),),
```

Comment:
Lint is failing, see the [contributing guide](https://github.com/pytorch/pytorch/blob/main/CONTRIBUTING.md#local-linting) on how to run lint locally.